web-console/package-lock.json

```diff
@@ -3274,12 +3274,14 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
```

Comment:
What's with the changes to this file, they seem unrelated?

Reply:
Oops, seems to be my mistake, thank you for the catch.